synapse/handlers/auth.py

```diff
@@ -1091,12 +1071,12 @@ def complete_sso_ui_auth(
         self._save_session(sess)
 
         # Render the HTML and return.
-        html_bytes = SUCCESS_TEMPLATE.encode("utf8")
+        html = self._sso_auth_success_template.encode("utf-8")
```

Comment:
Standard for the codebase seems to be calling this `html_bytes`, because it is a list of bytes.

But I'm open to opinions on making it cleaner everywhere.

Reply:
The two methods below this (`complete_sso_login` and `_complete_sso_login`) both call this `html` for what it's worth. Happy to change it (all of them?)

Reply:
Sure, go ahead

Reply:
Done!